server/rpc.go

```diff
@@ -367,14 +372,15 @@ func getOrchestrator(orch Orchestrator, req *net.OrchestratorRequest) (*net.Orch
 		return orchestratorInfo(orch, addr, orch.ServiceURI().String(), "")
 	}
 
-	if err := checkLiveVideoToVideoCapacity(orch, req.Capabilities); err != nil {
+	if err := checkLiveVideoToVideoCapacity(orch, req); err != nil {
 		return nil, fmt.Errorf("Invalid orchestrator request: %v", err)
 	}
 	return orchestratorInfoWithCaps(orch, addr, orch.ServiceURI().String(), "", req.Capabilities)
 }
 
-func checkLiveVideoToVideoCapacity(orch Orchestrator, caps *net.Capabilities) interface{} {
-	if caps.Constraints == nil || caps.Constraints.PerCapability == nil {
+func checkLiveVideoToVideoCapacity(orch Orchestrator, req *net.OrchestratorRequest) interface{} {
+	caps := req.Capabilities
+	if req.IgnoreCapacityCheck || caps.Constraints == nil || caps.Constraints.PerCapability == nil {
```

Comment:
This is the main change here, so if in the GetOrchestratorInfo request we include this flag, then we don't check the capacity.